### DISCUSSION TITLE
Coverage map and fading effects

### DISCUSSION THREAD

**Discussion by DrEStaudinger** [NVlabs/sionna#645] [Q&A] [answered]
Hi everybody,

I had a look at the coverage map tutorial and documentation and equation (52): https://nvlabs.github.io/sionna/api/rt.html#equation-cm-def

Maybe I missed something, but why do you assume the absolute value of h(s). Do you assume infinite signal bandwidth for the coverage map, or the other way around that you assume to resolve all received multipath components adequately?
The reason I am asking is, that I do not see any spatial fading effects in your coverage maps, as would naturally occur for bandwidth-limited signals, e.g. something like the 2-ray ground reflection model if I assume a single plane without obstacles and vertical polarization for transmitter and receiver.

As I read somewhere in an older post you assume flat ground with some obstacles for the coverage map? No actual terrain can be used with this particular functionality? If so, it would be great to have a hint in the documentation.

Best regards,
Emanuel

**Comment by jhoydis** [comment]
Hi @DrEStaudinger,

By definition, the coverage map averages fast-fading effects out by computing the energy of the non-coherently combined channel taps. If you need to see fast-fading effects, you can simply compute the exact channel impulse responses at the desired locations.

A coverage map can be computed for any type of terrain. However, it is always defined as a plane of arbitrary size and orientation that is discretized into small cells.

**Reply by DrEStaudinger** [reply]
Hi @jhoydis ,

Thank you for the explanation. May I kindly ask to add this information to the documentation as well?

I did not get your second explanation about the type of terrain yet. So you assume a single plane of certain size and orientation for the overall coverage map, e.g., the one you compute for Munich downtown? If I load a real digital elevation map (DEM), with a hilly terrain, would this still be taken properly into account for the coverage map functionality? So for example, the DEM being descretized into cells of 1m times 1m and assuming that each cell is then a plane with certain height and orientation?

Best regards,
Emanuel

**Comment by jhoydis** [comment, marked as answer]
Yes, we could add this to the documentation.

I guess you want to know if one could define a coverage map where each cell has the same height with respect to the ground. This is not the case. The entire coverage map is planar.
